pkg/gui/branches_panel.go

```diff
@@ -124,6 +124,32 @@ func (gui *Gui) handleCreatePullRequestPress(g *gocui.Gui, v *gocui.View) error
 	return nil
 }
 
+func (gui *Gui) handleGitFetch(g *gocui.Gui, v *gocui.View) error {
+	if err := gui.createMessagePanel(g, v, "", gui.Tr.SLocalize("FetchWait")); err != nil {
+		return err
+	}
+	go func() {
+		unamePassOpend, err := gui.fetch(g, v, true)
+		if err != nil {
+			errMessage := err.Error()
+			if errMessage == "exit status 128" {
```

Comment:
we're checking if the error message contains `exit status 128` in the `DetectUnamePass` and then we're checking if the string equals `exit status 128` here. Perhaps we should skip the check in `DetectUnamePass` and then in this function check if the string contains `exit status 128`

Reply:
That would be better but in that case i would need to choose between either sending an English return message or return error codes. I could also past the error code before the error message but that would not be great for the end user because they would see an error code if git errors on something else as wrong credentials